packages/optimizer/README.md

```diff
@@ -89,6 +89,12 @@ ampOptimizer.transformHtml(originalHtml, {
 })
 ```
 
+## Best Practices
+
+### Regenerate pages at least once a week
+
+AMP Optimizer inlines CSS styles required by AMP. To make sure, that the inlined CSS stays in sync with the latest AMP release, we recommend to re-generate pages at least once a week. The good news is, out-of-sync CSS will not break your page, as AMP will check the version of the inlined CSS at runtime and will automatically update it to the latest version.
```

Comment:
Re "not break your page": it might be worth explaining what will happen in a bit more detail if the CSS is out of date--this makes it sound like there's very little downside to _not_ regenerating pages once a week. Perhaps:

> ... we recommend regenerating pages once a week. (Out-of-sync CSS will not completely break your page, but in rare circumstances it can cause AMP components to briefly appear with the "wrong" styles, such as being visible when they should be hidden. The good news is that these glitches will only be temporary, because as soon as the AMP JS starts, it will check the inlined CSS and update it if required.)

Reply:
makes sense